src/main/java/uk/gov/pay/products/validations/GatewayAccountRequestValidator.java

```diff
@@ -26,7 +35,27 @@ public GatewayAccountRequestValidator(RequestValidations requestValidations) {
                 FIELD_OPERATION,
                 FIELD_OPERATION_PATH,
                 FIELD_VALUE);
+        if(errors.isPresent()) {
+            return errors.map(Errors::from);
+        }
+
+        return validateServiceNameRequest(payload).map(Errors::from);
+    }
 
-        return errors.map(Errors::from);
+    private Optional<List<String>> validateServiceNameRequest(JsonNode payload){
+        if(!payload.findValue(FIELD_OPERATION_PATH).asText().equals(FIELD_SERVICE_NAME)) {
+            return  Optional.of(asList(format("Operation [%s] not supported for path [%s]",
+                    FIELD_OPERATION,
+                    payload.findValue(FIELD_OPERATION_PATH).asText())));
+        }
+        String op = payload.get(FIELD_OPERATION).asText();
+        if (!VALID_ATTRIBUTE_UPDATE_OPERATIONS.get(FIELD_OPERATION).contains(op)) {
+            return Optional.of(asList(format("Operation [%s] is not valid for path [%s]", op, FIELD_OPERATION)));
+        }
```

Comment:
I don't really understand this if condition and the above if condition. Maybe the first error message should be "Path %s not supported / invalid"